mockserver.js

```diff
@@ -131,9 +131,14 @@ var mockserver = {
         }
 
         var watchedHeaders = module.exports.headers;
+
         if(watchedHeaders && !Array.isArray(watchedHeaders)) {
             watchedHeaders = [watchedHeaders];
         }
+        if(!watchedHeaders || !watchedHeaders.length) {
+            watchedHeaders = ['authorization']
+        }
```

Comment:
Then I would suggest to migrate it to a CLI option, as I found myself to have custom header support, I had to reimplement bin file with overrides. That should come out of the box, imo